eth_bridge/ic/src/eth_proxy/src/api/withdraw.rs

```diff
@@ -0,0 +1,51 @@
+use crate::common::weth::Weth;
+use crate::{
+    common::{
+        tera::Tera,
+        types::{EthereumAddr, TokendId, TxError, TxReceipt},
+    },
+    proxy::{ToNat, STATE, TERA_ADDRESS, WETH_ADDRESS_ETH},
+};
+use ic_kit::{
+    candid::{candid_method, Nat},
+    ic,
+    macros::update,
+    Principal,
+};
+
+/// withdraw left over balance if burn/mint fails
+/// this will attempt to bridge the leftover balance
+/// todo withdraw specific balance
+#[update(name = "withdraw")]
+#[candid_method(update, rename = "withdraw")]
+pub async fn withdraw(token_id: TokendId, eth_addr: EthereumAddr, _amount: Nat) -> TxReceipt {
```

Comment:
Same comment as before, pull from `feat/eth_proxy_with_state` and use hardcode eth address